OgreMain/include/OgreRenderTarget.h

```diff
@@ -218,7 +218,7 @@ namespace Ogre {
 
             | Name        | Description                        |
             |-------------|------------------------------------|
-            | WINDOW      | The native window handle. (X11 Window XID/ HWND / NSWindow*) |
+            | WINDOW      | The native window handle. (Wayland wl_egl_window* /X11 Window XID/ HWND / NSWindow*) |
```

Comment:
does it make sense to expose wl_egl_window*? Also you named it WL_WINDOW below

Reply:
The `WINDOW` parameter is only used in WindowEventUtilities. As long as we do not need to expose this for users of Ogre, there is no point in exposing `WINDOW` or any other pointers for that matter. I removed them in the latest commit.